galaxy/api/v2/tests/test_collection_view.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from galaxy.main import models
+
+
+class CollectionViewTests(TestCase):
+
+    def setUp(self):
```

Comment:
parent class setUp() method call is missing

Reply:
I can add that a `super().setUp()` here, but its not conventionally used in `setUp()` methods as they are explicitly hooks for the child class the parent does not provide any default behavior for, by definition.

Reply:
It's true in case of django TestCase hierarchy, but for instance DRF and django base classes override `setUpClass()` method, so IMO it's good practice to call parent class `setUp()` in case there is any logic present.
Also it might be useful for generic parent classes that provide common mocks for example.